skimage/segmentation/_join.py

```diff
@@ -131,14 +131,11 @@ def relabel_sequential(label_field, offset=1):
     required_type = np.min_scalar_type(offset + len(labels0))
     if np.dtype(required_type).itemsize > np.dtype(label_field.dtype).itemsize:
         label_field = label_field.astype(required_type)
-    new_labels0 = np.arange(offset, offset + len(labels0))
-    if np.all(labels0 == new_labels0):
-        return label_field, labels, labels
+    new_m = offset - 1 + len(labels0)
```

Comment:
Is this short for `new_maximum` or `new_max_label`? If so, I wouldn't mind this being the new name rather than `new_m`.

Reply:
> Is this short for `new_maximum` or `new_max_label`?

Yes. After relabeling, `relabeled.max() == new_m`.

> If so, I wouldn't mind this being the new name rather than `new_m`.

I personally don't have an opinion on how to call this variable. I just chose the name because `m` is already used:

https://github.com/scikit-image/scikit-image/blob/7d70b5e7dd1f5e33326cb50b339aa0b5c13073c4/skimage/segmentation/_join.py#L124

Reply:
@uschmidt83 [who would write such code???](https://github.com/scikit-image/scikit-image/commit/dc377970f520f8f04a2b3d879983c3a1171518e7#diff-09660f97c6b4c3dc5d143d3be12e8aaaR85) :joy: :joy: :joy: 

Well, our preference has evolved to more self-explanatory variable names, so I would support a renaming of `m` to `max_label` and `new_m` to `new_max_label`. Feel free to leave that one to us, though, as indeed it is an orthogonal change to your contribution. Thank you for this fix!

Reply:
I just changed the variable names as requested.